src/lib/navigation/strict-navigation-mode.spec.ts

```diff
@@ -243,5 +238,6 @@ describe('StrictNavigationMode', () => {
     expect(wizard.getStepAtIndex(2).selected).toBe(false);
     expect(wizard.getStepAtIndex(2).completed).toBe(false);
     expect(wizard.completed).toBe(false);
+    */
```

Comment:
Behavior of angular-archwizard changes, see discussion on the implementation of `ConfigurableNavigationMode.ensureCanReset` (https://github.com/madoar/angular-archwizard/pull/211/files#r283083971).

Reply:
I think the link to the discussion is wrong

Reply:
Sorry. Fixed.

Reply:
Checks removed in 15271ab